src/NServiceBus.Core.Tests/Serializers/Json/JsonMessageSerializerTest.cs

```diff
@@ -9,8 +9,33 @@ namespace NServiceBus.Serializers.Json.Tests
     using MessageInterfaces.MessageMapper.Reflection;
     using NUnit.Framework;
 
+    class MessageWithException
+    {
+        public Exception Blah { get; set; }
```

Comment:
I'm wondering whether we should make the issue with Exception more explicit instead of just using Exception. Afaik, any concrete class containing an interface would be an issue, did I get that right?